Directory.Build.props

```diff
@@ -1,35 +1,36 @@
 <Project>
-
   <PropertyGroup>
-    <_IsGitHub Condition="'$(GITHUB_ACTIONS)' == 'true'">true</_IsGitHub>
-    <_ContinuousIntegrationBuild Condition="'$(TF_BUILD)' == 'true' or '$(GITHUB_ACTIONS)' == 'true' or '$(GITLAB_CI)' == 'true'">true</_ContinuousIntegrationBuild>
-
     <!-- Assembly Info -->
     <Company>Ubisoft</Company>
     <Copyright>Copyright © $([System.DateTime]::UtcNow.Year) $(Company)</Copyright>
     <Description>GitLab REST API .NET Client Library</Description>
     <Product>NGitLab</Product>
+
     <!-- Compile Options -->
     <LangVersion>9.0</LangVersion>
     <Deterministic>true</Deterministic>
     <Features>strict</Features>
     <TreatWarningsAsErrors Condition="'$(Configuration)' != 'Debug'">true</TreatWarningsAsErrors>
 
     <RestoreUseStaticGraphEvaluation>true</RestoreUseStaticGraphEvaluation>
+	
+	<!-- Package info -->
```

Comment:
Replace TABs with spaces throughout the file.

Reply:
done